File: BookReview.API/Models/Chapter.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BookReview.API.Models
{
    public class Chapter
    {
        public int Id { get; set; }
        public virtual Book Book { get; set; }
        public int BookId { get; set; }
        public virtual User User { get; set; }
        public int UserId { get; set; }
        public string ChapterName { get; set; }
        public string Content { get; set; }
    }
}

File: BookReviewApp-SPA/src/app/_models/chapter.ts

/**
 * @description Interface that represents a Chapter Entity
 */
export interface Chapter {
  /**
   * @param id the Id of the Chapter
   */
  id: number;

  /**
   * @param id the Id of the Book
   */
  bookId: number;

  /**
   * @param id the Id of the User
   */
  userId: number;

  /**
   * @param id the Name of the Chapter
   */
  chapterName: string;

  /**
   * @param id the Content of the Chapter
   */
  content: string;
}
